### DISCUSSION TITLE
Add ability to control FBW A32NX (Dev version) custom autopilot

### DISCUSSION THREAD

**Discussion by alvoram** [nguyenquyhy/Flight-Tracker-StreamDeck#240] [Q&A]
Would it be possible to enable us to trigger the flybywire a320nx community mods new custom auto pilot events...

Found here https://github.com/flybywiresim/a32nx/blob/master/docs/a320-events.md

The one I'm really desperate for is A32NX.FCU_ALT_PULL

Previous versions of the fbw mod, and the default A320, used FLC to achieve this, sadly, this no longer works at all :(

If this is already possible, can somebody tell me how :(

**Comment by nguyenquyhy** [comment]
For now the plugin communicates with the sim only via SimConnect, which isn't get much support from FlyByWire. The workaround is that you can try to use MobiFlight as a bridge. MobiFlight WASM module convert local events into SimConnect events, which can be triggered by the plugin.

If MobiFlight does support the event you need, please file a new Issue with the list of MobiFlight event and I'll add them in next update. We have already been doing that for the CJ4.
